.github/workflows/update-cert.yaml

```
@@ -36,10 +36,5 @@ jobs:
         git status
         DATE="$(date)"
         export DATE
-        if [ "$(git diff-index --quiet HEAD)" ]; then
-          echo "cacert.pem needs to be updated" &&
-          git add -a &&
-          git commit -m "cacert.pem updated $DATE" &&
-          git push origin main
-        fi
+        ( git commit -am "cacert.pem updated $DATE" && git push origin main ) || true
```

A bit safer - more targeted
```suggestion
        ( git commit cacert.pem -am "cacert.pem updated $DATE" && git push origin main ) || true
```